components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -2485,135 +2485,140 @@ public static NativeArray jsFunction_getAllSubscriptions(Context cx,
 
             Subscriber subscriber = new Subscriber(username);
             APIConsumer apiConsumer = getAPIConsumer(thisObj);
-            Application[] applications = apiConsumer.getApplications(new Subscriber(username));
-            if (applications != null) {
+            //Application[] applications = apiConsumer.getApplications(new Subscriber(username));
```

Comment:
The content of this method is too long, it should be refactored to split the logic into several methods. So it would be more readable.